ui/app/src/components/CreateTokenDialog/CreateTokenDialogUI.tsx

```diff
@@ -146,13 +144,7 @@ export function CreateTokenDialogUI(props: CreateTokenUIProps) {
         <SecondaryButton onClick={onDismiss}>
           <FormattedMessage id="words.cancel" defaultMessage="Cancel" />
         </SecondaryButton>
-        <PrimaryButton
-          type="submit"
-          onClick={onSubmit}
```

Comment:
Submit was already handled by tag, and being here caused the double submit/token creation